### ISSUE TITLE
Reproduce the paper's results 

### ISSUE THREAD

**Issue by L3viAckerman** [atomistic-machine-learning/schnetpack#207]
Hi @ktschuett , I am trying to reproduce the paper results too, but I'm facing with that time need for training model is too much, about 130s for one epoch. So when I read the params, I see epochs is 5000, so huge for my device (I only have 1 K40m GPU). I wonder what number of epochs is the min for training model to reproduce the paper results. Thanks you so much!

**Comment by Stefaanhess** [comment]
Hi @L3viAckerman ,

Reproducing the benchmark results for QM9 takes about 1-2 minutes per epoch. So the 130s seems to be reasonable. I am not sure how many epochs it takes until the model converges to the banchmark values, but it will never take 5000 epochs. The *epochs* parameter sets the maximum number of epochs. Since the training script uses a learning rate scheme by default, the training stops automatically when the model has converged.
If you are only interested in a trained model, you could also download a trained model from https://schnetpack.readthedocs.io/en/stable/getstarted/benchmarks.html .

Best regards,
Stefaan

**Comment by L3viAckerman** [comment]
Hi @Stefaanhess ,
I am a student and I am doing my thesis about Machine Learning in Quantum Chemistry, so reproducing Schnet paper's results is a part of my work.
Thank you so much for helping me and wish the best for your and your team in 2020.

Best regards,
LinhLPV

**Comment by Stefaanhess** [comment]
Thanks and good luck!
